core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/AutoTuner.scala

```diff
@@ -264,6 +264,110 @@ class RecommendationEntry(val name: String,
   }
 }
 
+// scalastyle:off line.size.limit
+/**
+ * Resolves the appropriate RapidsShuffleManager class name based on Spark or Databricks version.
+ *
+ * Note:
+ * - Supported RapidsShuffleManagers: https://docs.nvidia.com/spark-rapids/user-guide/latest/additional-functionality/rapids-shuffle.html#rapids-shuffle-manager
+ * - Version mappings need to be updated as new versions are supported.
+ * - This can be extended to support more version mappings (e.g. Cloudera).
+ */
+// scalastyle:on line.size.limit
+object ShuffleManagerResolver {
```

Comment:
This is kinda platform specific thing, right. Why cannot we have it in the Platform class?

Reply:
Yes. I will refactor this to make it platform specific.